include/asset/asset_ticker.hpp

```diff
@@ -0,0 +1,19 @@
+#pragma once
+
+#include <string>
+
+class AssetTicker {
+ public:
+  explicit AssetTicker(const std::string& ticker) noexcept;
```

Comment:
Please, drop `explicit`. Precisely, the duplication of operators is relying on implicit constructor

```cpp
bool operator==(const AssetTicker& ticker) const;
bool operator==(const std::string& ticker) const;
```